src/metabase/models/field_values.clj

```diff
@@ -440,9 +440,19 @@
                     ;; It's too short, so no schema. Shift them over and add a nil schema.
                     [db nil schema table])]
     (-> (serdes.base/load-xform-basics fv)
-        (assoc :field_id (serdes.util/import-field-fk field-ref)))))
+        (assoc :field_id (serdes.util/import-field-fk field-ref))
+        (update :type keyword))))
 
 (defmethod serdes.base/load-find-local "FieldValues" [path]
   ;; Delegate to finding the parent Field, then look up its corresponding FieldValues.
   (let [field-id (serdes.base/load-find-local (pop path))]
     (db/select-one-id FieldValues :field_id field-id)))
+
+(defmethod serdes.base/load-update! "FieldValues" [_ ingested local]
```

Comment:
This is an aside, but at some point we should probably change these to dispatch off of the actual model e.g. `FieldValues` instead of the model name at some point. See `serdes.hash/identity-hash-fields` for example. The main benefit is that the actual models can go in a hierarchy so we can share method impls where feasible, sort of like we do with the perms stuff these days

```clj
(doto Collection
  (derive ::mi/read-policy.full-perms-for-perms-set)
  (derive ::mi/write-policy.full-perms-for-perms-set))
```

When we switch to Toucan 2 and `FieldValues` becomes the keyword `:metabase.models.field-values/FieldValues` I guess `name` would still work, since `(name :metabase.models.field-values/FieldValues)` is still `"FieldValues"`, but I don't plan on making `(name some-instance-of-a-field-values)` work. But `(toucan2.core/model some-instance-of-a-field-values)` will return `:metabase.models.field-values/FieldValues`, so if we're dispatching off of _model_ already then we won't need to change anything.